docs/telemetry.md

```diff
@@ -1,32 +1,29 @@
 # Telemetry
 
-To improve the end-user experience, Monkle collects anonymous data about usage.
+To improve the end-user experience, Monokle collects telemetry data.
 
 Participation in this program is optional, and you may [opt-out](#how-to-opt-out) if you'd prefer not to share any information.
 
-This data is always anonymous to gather numbers of users and how they are using Monokle. The data collected is not traceable to the source and only used in its aggregate form.
-
-The data collected is always anonymous, not traceable to the source, and only used in aggregate form. 
+The data collected is always anonymous, not traceable to the source, and only used in aggregate form. You can learn more how we treat your data by reading our [privacy statement](https://monokle.kubeshop.io). 
```

Comment:
You can learn more **about** how we treat your data

Reply:
And I do not see a privacy statement on the Monokle web site when I click the link.

Reply:
The link is not there yet, it's a placeholder for the definite URL.